### PR TITLE
[verisure] Verisure Binding initial contribution

### PR DESCRIPTION
**Pull request by jarlebh**
Verisure Binding

Initial commit of the Versiure Binding

This has already appeared as a pull request before, recreated with my own user as the pull initiatior
Replaces #2942

Jarle Hjortland

### CODE REVIEW

**Comment by jarlebh** [comment]
I have added the signoff

File: addons/binding/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/handler/VerisureBridgeHandler.java [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+            cuid = new ChannelUID(getThing().getUID(), CHANNEL_CHANGERNAME);
+            updateState(cuid, session.getAlarmChangerName());
+
+            cuid = new ChannelUID(getThing().getUID(), CHANNEL_TIMESTAMP);
+            updateState(cuid, session.getAlarmTimestamp());
+
+            cuid = new ChannelUID(getThing().getUID(), CHANNEL_STATUS_LOCALIZED);
+            updateState(cuid, session.getAlarmObject() == null ? UnDefType.UNDEF
+                    : new StringType(session.getAlarmObject().getLabel()));
+        } catch (Exception e) {
```
Comment (wborn) [inline addons/binding/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/handler/VerisureBridgeHandler.java]: Please narrow the caught exception to only those types of exceptions that are actually thrown.

File: addons/binding/org.openhab.binding.verisure/README.md [hunk 2]
```diff
@@ -43,10 +43,10 @@ Lock ([lock]) support the following channels:
 <table>
 <tr><td><b>Channel Type ID</b></td> <td><b>Item Type</b></td> <td><b>Description</b></td> </tr>
 <tr><td>status</td><td>String</td><td>This channel reports the lock status.</td></tr>
-<tr><td>statusnumeric</td><td>Number</td><td>This channel reports the lock status as a number.</td></tr>
 <tr><td>timestamp</td><td>String</td><td>This channel reports the last time lock was changed (not a Date).</td></tr>
 <tr><td>changername</td><td>String</td><td>This channel reports the username that changed the state of the lock.</td></tr>
-<tr><td>setstatus</td><td>Number</td><td>This channel is used to arm/disarm the lock.</td></tr>
+<tr><td>location</td><td>String</td><td>This channel reports the location fo lock as setup in Verisure.</td></tr>
```
Comment (robinsving) [inline addons/binding/org.openhab.binding.verisure/README.md]: "location of"

**Comment by Hilbrand** [comment]
This pull request is replaced by pr #4789 and therefor will be closed. See mentioned pr for more information.

**Pull request closed without merging**